test/performance/odp_packet_gen.c

```diff
@@ -252,15 +260,18 @@ static void print_usage(void)
 	       "                              num_tx * burst_size * bursts * (10^9 / gap)\n"
 	       "  -s, --ipv4_src            IPv4 source address. Default: 192.168.0.1\n"
 	       "  -d, --ipv4_dst            IPv4 destination address. Default: 192.168.0.2\n"
-	       "  -o, --udp_src             UDP source port. Default: 10000\n"
-	       "  -p, --udp_dst             UDP destination port. Default: 20000\n"
+	       "  -o, --src_port            Source port. Default: 10000\n"
+	       "  -p, --dst_port            Destination port. Default: 20000\n"
+	       "  -N, --proto               L4 protocol. Default: 0\n"
```

Comment:
With additional protocols, might be useful to somehow bind different features to the protocols that actually support them. E.g., port number increasing only for protocols that use ports etc. Would save option namespace and probably improve usability. Anyways, out of the context of this PR, maybe a future improvement.